src/tax-lot/tax-lot.repository.schema.ts

```diff
@@ -0,0 +1,50 @@
+import { z } from "zod";
+import { boroughEntitySchema, landUseEntitySchema } from "src/schema";
+
+export const findByBblSchema = z.object({
+  bbl: z.string().regex(RegExp("^([0-9]{10})$")),
+  block: z.string().regex(RegExp("^([0-9]{1,5})$")),
+  lot: z.string().regex(RegExp("^([0-9]{1,4})$")),
+  address: z.string().nullable(),
+  borough: boroughEntitySchema,
+  landUse: landUseEntitySchema.nullable(),
+});
+
+export type FindByBbl = z.infer<typeof findByBblSchema>;
+
+export const findByBblSpatialSchema = findByBblSchema.extend({
+  geometry: z
+    .string()
+    /**
```

Comment:
Re: our disc. we'll put this in a reusable section once we build out testing more.